src/main/resources/application.yml

```diff
@@ -19,9 +19,14 @@ spring:
     password: nakadi
     driverClassName: org.postgresql.Driver
     initialSize: 2
-    maxActive: 20
+    maxActive: 2
```

Comment:
this property will be overridden with senza env variables we have in our private repo